logging/tests/unit/handlers/transports/test_background_thread.py

```diff
@@ -282,6 +284,22 @@ def test__thread_main(self):
         self.assertEqual(worker._cloud_logger._batch.commit_count, 2)
         self.assertEqual(worker._queue.qsize(), 0)
 
+    def test__thread_main_no_python_logger(self):
+        from google.cloud.logging.handlers.transports import background_thread
+
+        worker = self._make_one(_Logger(self.NAME), include_logger_name=False)
+
+        # Enqueue one record and the termination signal.
+        self._enqueue_record(worker, '1')
+        worker._queue.put_nowait(background_thread._WORKER_TERMINATOR)
+
+        worker._thread_main()
+
+        self.assertEqual(len(worker._cloud_logger._batch.all_entries), 1)
+        self.assertFalse(
+            'python_logger' in worker._cloud_logger._batch.all_entries[0]
+        )
```

Comment:
Why the added `all_entries` bit?  Isn't `entries` going to be the same?  The comment below says "All entries ever committed via this _Batch", but the code only mimics what is added to `self.entries` in `log_struct` (nothing in `commit`).

Reply:
@tseaver because .entries gets purged in the commit function below, and all_entries is not purged. Adding all_entries lets us test that the entries were emitted.

Reply:
OK